src/wallet/feebumper.cpp

```diff
@@ -152,8 +152,14 @@ bool TransactionCanBeBumped(const CWallet& wallet, const uint256& txid)
 }
 
 Result CreateRateBumpTransaction(CWallet& wallet, const uint256& txid, const CCoinControl& coin_control, std::vector<bilingual_str>& errors,
-                                 CAmount& old_fee, CAmount& new_fee, CMutableTransaction& mtx, bool require_mine, const std::vector<CTxOut>& outputs)
+                                 CAmount& old_fee, CAmount& new_fee, CMutableTransaction& mtx, bool require_mine, const std::vector<CTxOut>& outputs, std::optional<uint32_t> reduce_output)
 {
+    // Cannot both specify new outputs and an output to reduce
+    if (!outputs.empty() && reduce_output.has_value()) {
```

Comment:
Is this not redundant since already checked in spend.cpp? before calling `CreateRateBumpTransaction`?

Reply:
Could we perhaps consider relying on the check here? It seems like a duplicate of the same process in two different places if you agree.

Reply:
Done